.github/workflows/integration-informing.yaml

```diff
@@ -76,29 +76,36 @@ jobs:
         uses: actions/download-artifact@v4
         with:
           name: k8s-${{ matrix.patch }}.snap
-          path: build
+          path: ${{ github.workspace }}/build
       - name: Apply ${{ matrix.patch }} patch
         run: |
           ./build-scripts/patches/${{ matrix.patch }}/apply
       - name: Run end to end tests
+        env:
+          TEST_SNAP: ${{ github.workspace }}/build/k8s-${{ matrix.patch }}.snap
+          TEST_SUBSTRATE: lxd
+          TEST_LXD_IMAGE: ${{ matrix.os }}
+          TEST_INSPECTION_REPORTS_DIR: ${{ github.workspace }}/inspection-reports
+          # TODO(ben): Use candidate/stable
+          # -strict suffix is removed from the channel name in the run script
+          TEST_VERSION_UPGRADE_CHANNELS: |
+              1.30-${{ matrix.patch }}/edge
+              1.31-${{ matrix.patch }}/edge
```

Comment:
i guess you'd rather have these as the "previous 2" rather than the fix 1.30 and 1.31?